modules/spo-binding-workloads.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-binding-workloads_{context}"]
+= Binding workloads to profiles with ProfileBindings
+
+You can use the `ProfileBinding` resource to bind a security profile to the `SecurityContext` of a container.
+
+.Procedure
+
+. To bind a `Pod` that uses an `nginx:1.19.1` image to the example `{kind}` profile, create a `ProfileBinding` object in the same namespace with the `Pod` and the `{kind}` objects:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: security-profiles-operator.x-k8s.io/v1alpha1
+kind: ProfileBinding
+metadata:
+  namespace: my-namespace
+  name: nginx-binding
+spec:
+  profileRef:
+    kind: {kind}
+    name: profile
+  image: nginx:1.19.1
+----
+
+. Label the namespace to `enable-binding=true` by running the following command:
++
+[source,terminal]
+----
+$ oc label ns spo-test spo.x-k8s.io/enable-binding=true
+----
+
+. Delete and re-create the pod to use the `ProfileBinding` object:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc delete pods test-pod && oc create -f pod01.yaml
+----
+
+.Verification
+
+ifdef::seccomp[]
+. Confirm the pod inherits the `ProfileBinding` by running the following command:
++
+[source,terminal]
+----
+$ oc get pod test-pod -o jsonpath='{.spec.containers[*].securityContext.seccompProfile}'
+----
++
+.Example output
+[source,terminal]
+----
+{"localhostProfile":"operator/my-namespace/profile-complain-unsafe.json","type":"Localhost"}
```

Comment:
to align with above profilebinding profile, the output name should be
`{"localhostProfile":"operator/my-namespace/profile.json","type":"Localhost"}`